.github/workflows/_release.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         working-directory: ${{ inputs.working-directory }}
 
       - name: Upload build
-        uses: actions/upload-artifact@v3
```

Comment:
Changes from [this PR](https://github.com/langchain-ai/langchain-unstructured/pull/5/files) never passed CI and were not added to main. Adding them in this PR now that CI is passing.